examples/mixer_demo.rs

```diff
@@ -11,10 +11,12 @@ fn main() {
 
     let args: Vec<_> = env::args().collect();
 
-    if args.len() < 3 {
-        println!("Usage: ./demo music.[mp3|wav|ogg] sound-effect.[mp3|wav|ogg]")
-    } else {
+    if args.len() == 2 {
+        demo(Path::new(&args[1]), Path::new(""));
+    } else if args.len() == 3 {
```

Comment:
Trying to use `.get(2)` directly as the input seems to not work since we need to re-wrap into a new `Option<T>`.

```
25 |         demo(Path::new(&args[1]), args.get(2));
   |                                   ^^^^^^^^^^^ expected struct `std::path::Path`, found struct `std::string::String`
   |
   = note: expected type `std::option::Option<&std::path::Path>`
              found type `std::option::Option<&std::string::String>`
```

Let me know if there is a more idiomatic way to do this than what I did, which was to de-structure and re-structure.